docs/I7solrImporter.md

```diff
@@ -87,26 +87,21 @@ Select the data transformation approach--how your source data will be transforme
 
 - Lastly, for this step, you will need to select the destination Fields and Bundles for your New ADOs. If your spreadsheet source only contains Digital Objects, select `Strawberry (Descriptive Metadata source) for Digital Object`
 
-    - If using Sheet 1 of the Demo AMI Ingest set (found above):
-        - Select `Template` and use the AMI Ingest JSON template that corresponds with your metadata elements.
-        - Select `images`, `documents`, and `audios` for the file source/fetching.
+    - Select `Template` and use the AMI Ingest JSON template that corresponds with your metadata elements.
+    - Select `images`, `documents`, and `audios` for the file source/fetching.
 
              ![Islandora Solr Importer Data Transformation](images/I7step3_DataTransformation_updated_2022-11.jpg)
 
 ## Step 4: Global ADO Mappings
 
 Select your global ADO mappings.
 
-- Even if empty (no values), select `node_uuid` and any relationship predicate columns (such as `ismemberof`).
-- By default, the option to automatically assigns UUIDs is selected. If you have existing UUIds, unselect this option.
+- Even if empty (no values), select any relationship predicate columns (such as `ismemberof` and `ispartof`, if present) for ADO Parent columns.
```

Comment:
Could you (sorry 😢 ) make sure the screenshot also show `ispartof` selected?

Reply:
Ah, I thought the same thing, but that might be challenging--since I don't have a legacy I7 collection with known compounds to reference at the moment (all have been migrated already 😸 )

Reply:
I can manually make one using a regular AMI set screenshot though. Give me a few minutes.

Reply:
Ok, all good now @DiegoPino ?

Reply:
Yes! Thanks so much